src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -1068,3 +1068,17 @@ PARAM_DEFINE_INT32(FW_LND_NUDGE, 0);
  * @group FW L1 Control
  */
 PARAM_DEFINE_INT32(FW_LND_ABORT, 0);
+
```

Comment:
I purposely did not put all of that logic into the FW_LND_USETER description, because all that param does is say use the distance sensor - it does not specify what abort logic is used, or what FW_LND_TER_REL does. Those are independent. Could maybe just note there that if USETER = 0, then no terrain based abort logic is used, and no terrain following on glide slope is possible.

Reply:
>only used for flaring start if FW_LND_TER_REL ==1

Just fyi - It is opposite. FW_LND_TER_REL==1 means that we listen to the terrain estimate the entire approach. Which is why it is defaulted to 0 (so that we only use it for flare trigger).

>And I would make FW_LND_USETER 1 by default.

Just to confirm - this means that folks without distance sensors (putting all default settings) will still land without aborting at the land waypoint altitude (since the default FW_LND_ABORT bits are none). Maybe we want default FW_LND_ABORT bits to be all active if the distance sensor is enable? Then it will abort if no distance sensor exists, and the user needs to actively disable the unsafe things?

Reply:
I would though mention about flaring in the description of FW_LND_USETER, as this is the main reason to have the distance sensor enabled.

Reply:
And yes, I meant FW_LND_TER_REL==0 :+1:

Reply:
great!